infrastructure/loadtesting/terraform/readme.md

```diff
@@ -119,11 +119,12 @@ Following are the steps to deploy new code changes to osquery-perf (known as `lo
 > The following is to add new hosts with new/modified osquery-perf code. (This happens if during a load test
 > the developer realizes there's bug in osquery-perf or if it's not simulating osquery properly.)
 
-> You don't need to push code changes because the `loadtest` image is built locally from your fleet repository source code.
+> You must push your code changes to the `$BRANCH_NAME`.
```

Comment:
I thought you didn't need to push changes for osquery-perf updates, only for updates to fleet?

Reply:
I thought so, but it seems we need to push, see:
https://github.com/fleetdm/fleet/blob/e71cdf70e60f26265e233aa5fd92327f69e426cc/infrastructure/loadtesting/terraform/docker/loadtest.Dockerfile#L3

(I realized because my local changes weren't being deployed.)

Reply:
Aha, gotcha. Sounds good!